packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/cleanData.js

```diff
@@ -135,13 +135,11 @@ const cleanData = ({ browserState, serverState }, currentSchema, componentsSchem
           }, []);
 
           cleanedData = {
+            options: {
```

Comment:
just wondering, those changes don't seem correlated to the use callback ref doc 👀

Reply:
I changed the title, it's a bit of a joint PR – I know they should be separate PRs 😓